test_scripts/Defects/5_1/1004_App_is_disconnected_due_to_PROTOCOL_VIOLATION_when_start_audio_streaming_after_rejected_2_times_then_accepted.lua

```diff
@@ -0,0 +1,81 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1004
+-- Description: App is disconnected due to PROTOCOL_VIOLATION when start audio streaming after rejected 2 times then accepted
+-- Precondition:
+-- 1) SDL and HMI are started.
+-- 2) App is registered and activated
+-- In case:
+-- 1) Press "Audio Service - Show" button
+-- 2) Press "Start Service"
+-- 3) Press "Cancel" on pop-up
+-- 4) Press "Cancel" on pop-up
+-- 5) Press "OK" pop-up
+-- 6) Press "Start File Streaming"
+-- Expected result:
+-- 1) Audio Streaming is started.
+-- Actual result:
```

Comment:
Actual result is redundant, please remove

Reply:
@GetmanetsIrina You can find the updates in the commit 05f7cf9